src/server/common/helpers/redis-client.js

```diff
@@ -1,63 +1,44 @@
 import IoRedis from 'ioredis'
-
-import { config } from '~/src/config/index.js'
 import { createLogger } from '~/src/server/common/helpers/logging/logger.js'
 
 /**
  * Setup Redis and provide a redis client
  *
  * Local development - 1 Redis instance
  * Environments - Elasticache / Redis Cluster with username and password
- *
+ * @param {RedisConfig} redisConfig - Redis config
  * @returns {Cluster | Redis}
```

Comment:
Might be easier removing this temporarily until 

Currently the type `RedisConfig` isn't defined anywhere

```suggestion
```